### PR TITLE
Update ValidationStep to display the new Let's chat text and image

### PR DESCRIPTION
**Pull request by HorusGoul**
@kevinksullivan @shawnborton 

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/179254

### Tests
1. Create workspace
2. Go through VBA flow with [test credentials in (1)](https://stackoverflow.com/c/expensify/questions/342/525#525)
3. Arrive at ValidateStep. The title, image, text and CTA should be like the solution described in https://github.com/Expensify/Expensify/issues/179254

### QA Steps
1. Create workspace
2. Go through VBA flow with [test credentials in (1)](https://stackoverflow.com/c/expensify/questions/342/525#525)
3. Arrive at ValidateStep. The title, image, text and CTA should be like the solution described in https://github.com/Expensify/Expensify/issues/179254

### Tested On

- [x] Web
- [x] Mobile Web
- [ ] Desktop
- [ ] iOS
- [ ] Android

#### Web

localhost_8080_bank-account_

### CODE REVIEW

File: src/pages/ReimbursementAccount/ValidationStep.js [hunk 1]
```diff
@@ -165,10 +166,12 @@ class ValidationStep extends React.Component {
...
 
         const maxAttemptsReached = lodashGet(this.props, 'reimbursementAccount.maxAttemptsReached');
+        const isVerifying = !maxAttemptsReached && state === BankAccount.STATE.VERIFYING;
+
         return (
             <View style={[styles.flex1, styles.justifyContentBetween]}>
                 <HeaderWithCloseButton
-                    title={this.props.translate('validationStep.headerTitle')}
+                    title={isVerifying ? this.props.translate('validationStep.letsChatTitle') : this.props.translate('validationStep.headerTitle')}
                     stepCounter={{step: 5, total: 5}}
```
Comment (HorusGoul) [inline src/pages/ReimbursementAccount/ValidationStep.js:175]: Unsure about the `Step 5 of 5`. That didn't appear in the Figma file; should I remove it when displaying the `Let's chat!` title?

**Comment by shawnborton** [comment]
Looks good to me!

File: assets/images/lets-chat.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...0 0 125 59" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M94.5 25.7v7.9" stroke="#0B1B34" stroke-width="2" stroke-miterlimit="10"/><path d="M118.7 51c.1-11.1-10.8-20.2-24.2-20.2-13.4 0-24.3 9-24.2 20.2h48.4Z" fill="#0185FF" stroke="#fff" stroke-width="2" stroke-miterlimit="10"/><path d="M112.1 45.1c-1.6-3.4-4.7-6.2-8.5-8l8.5 8Z" fill="#0B1B34"/><path d="M112.1 45.1c-1.6-3.4-4.7-6.2-8.5-8" stroke="#fff" stroke-width="2" stroke-miterlimit="10" stroke-linecap="round" stroke-linejoin="round"/><path d="M98.8 26.7c0-2-2-3.6-4.4-3.6-2.4 0-4.4 1.6-4.4 3.6h8.8Z" fill="#0B1B34"/><path d="M122.4 58H66.5l3.7-6.9h48.5l3.7 6.9Z" fill="#0185FF" stroke="#fff" stroke-width="2" stroke-miterlimit="10"/><path d="M94.1 10.2v7.1M83 13.2l4 5.1M105.2 13.2l-4 5.1" stroke="#0B1B34" stroke-width="2" stroke-miterlimit="10"/><path d="M1.5 23.6c0 12.5 10.1 22.6 22.6 22.6 5.4 0 10.3-1.9 14.2-5h18.4L45.5 30.8c.8-2.3 1.2-4.7 1.2-7.2C46.7 11.1 36.6 1 24.1 1 11.6 1 1.5 11.1 1.5 23.6Z" fill="#0185FF" stroke="#fff" stroke-width="2" stroke-miterlimit="10" stroke-linejoin="round"/><path d="M21.9 11.6h4.6v5.7l-1.2 10.4h-2.2l-1.2-10.4v-5.7ZM24.2 34.5a2.3 2.3 0 1 0 0-4.6 2.3 2.3 0 0 0 0 4.6Z" fill="#fff"/></svg>
```
Comment (shawnborton) [inline assets/images/lets-chat.svg:1]: Just confirming that in my Figma mockup, I have this image at 160x100:
image

Just curious where this image is coming from as it seems like the size is different?

Comment (shawnborton) [reply assets/images/lets-chat.svg:1]: Here is my Figma version: [lets_chat.svg.zip](https://github.com/Expensify/App/files/7249513/lets_chat.svg.zip)

Comment (HorusGoul) [reply assets/images/lets-chat.svg:1]: I'll update that! I got it from Figma, but I selected the layer that doesn't have the spacing to export it.

File: src/pages/ReimbursementAccount/ValidationStep.js (hunk 1 above)
Comment (MariaHCD) [reply src/pages/ReimbursementAccount/ValidationStep.js:175]: I think it makes sense to keep it to remain consistent with the previous steps?

**Review by MariaHCD — APPROVED** [review]
LGTM!

File: src/pages/ReimbursementAccount/ValidationStep.js (hunk 1 above)
Comment (kevinksullivan) [reply src/pages/ReimbursementAccount/ValidationStep.js:175]: the step counter hit staging yesterday https://github.com/Expensify/App/pull/5255

**Review by kevinksullivan — APPROVED** [review]

**Comment by kevinksullivan** [comment]
Just want to clarify that this step will change slightly as soon as the doc for this [WN post ](https://expensify.slack.com/archives/CC7NECV4L/p1632861743162900?thread_ts=1632798461.112600&cid=CC7NECV4L)is getting implemented... I think it is fine to merge and move forward for now, but felt I should note that in case we think there is a reason to hold.

**Comment by isagoico** [comment]
https://github.com/Expensify/App/issues/5622 is failing this PR cc @HorusGoul

**Comment by isagoico** [comment]
Retested this and it was a pass 🎉 
image

**Pull request merged**